priv/gettext/zh_Hans/LC_MESSAGES/default.po

```diff
@@ -0,0 +1,573 @@
+## This file is a PO Template file.
+##
+## `msgid`s here are often extracted from source code.
+## Add new translations manually only if they're dynamic
+## translations that can't be statically extracted.
+##
+## Run `mix gettext.extract` to bring this file up to
+## date. Leave `msgstr`s empty as changing them here as no
+## effect: edit them in PO (`.po`) files instead.
+msgid ""
+msgstr ""
+"Language: zh_Hans\n"
```

Comment:
Here, an An IETF BCP 47 language tag is required, which is why I  replaced `chs` with `zh_Hans`. Hope that's okay :) The same applies to the folder name.

Reply:
Brilliant!

Reply:
How to set my panel to Chinese? Thank you!